src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/create_index_pattern_wizard.js

```diff
@@ -221,9 +221,7 @@ uiModules.get('apps/management')
       const dateFields = fields.filter(field => field.type === 'date');
 
       if (dateFields.length === 0) {
-        return [{
-          display: `The indices which match this index pattern don't contain any time fields.`,
```

Comment:
I think we want to keep this message, right? I think it's just the caret which is out of place, unless I'm missing something.

Reply:
This string isn't actually being used for the display 🙂 The real string is in `step_time_field.html`

Reply:
Oh wow OK!